mods/persistence/__defines/saved_var.dm

```diff
@@ -0,0 +1,21 @@
+/**
+ * Define a var from a class as saved. 
+ * Also test the CLASSPATH class to see if the var exist at compilation. 
+ */
+#define SAVED_VAR(CLASSPATH, VARNAME) CLASSPATH/proc/zz_compilerVarExistsTest_##VARNAME(){.=src.##VARNAME;}; /decl/saved_variables##CLASSPATH/make_saved_variables(list/saved){LAZYDISTINCTADD(saved, #VARNAME); return ..(saved);}
+
+/**
+ * Clears all saved variables for this particular type and its children.
```

Comment:
Might want to mention that it only saves its position/existence or whatever. Makes its purpose more explicit.

Reply:
I guess I wrote it at the wrong place. I'll add a copy in there.